lib/Heartbeat/HeartbeatFlashing.cpp

```diff
@@ -27,6 +27,8 @@
   SOFTWARE.
 */
 
+#ifdef HB_FLASHING
+
```

Comment:
Why we need this, where is it used?

Reply:
Platformio tries to build the object file but got somehow confused as this code is not used abywhere. So this is for now basically a `#if 0`.